README.md

```diff
@@ -1 +1,98 @@
 # Frames Adapter
```

Comment:
I believe this should be `frames-adapter` (to mirror the package name)

Reply:
Fixed